### ISSUE TITLE
WebTauUiFeaturesTest failures

### ISSUE THREAD

**Issue by tsiq-karold** [twosigma/webtau#320]
I'm getting the following failure:
```
[ERROR] Failures:
[ERROR]   WebTauUiFeaturesTest.forms:162->runCli:256 matches:

testDetails.exitCode:   actual: 1 <java.lang.Integer>
                      expected: 0 <java.lang.Integer>
testDetails.scenarioDetails[4].stepsSummary.numberOfSuccessful:   actual: 2 <java.lang.Integer>
                                                                expected: 3 <java.lang.Integer>

extra values:

testDetails.scenarioDetails[4].stepsSummary.numberOfFailed: 1
[INFO]
[ERROR] Tests run: 40, Failures: 1, Errors: 0, Skipped: 0
```

Running the forms test locally, I see this in the report:
```
java.lang.AssertionError: 
mismatches:

value:   actual: "12016-02-06" <java.lang.String>
       expected: "2016-06-21" <java.lang.String>
```

The assertion that produces this is:
```groovy
scenario('values validation') {
    form.name.should == 'Full Automation'
    form.rank.should == 'B'
    form.startDate.should == '2016-06-21'
}
```

Could this be a locale issue?  The start date is set as follows:
```groovy
form.startDate.setValue('06/21/2016')
```

On a London laptop that probably results in it setting it to the 21st month...

**Comment by MykolaGolubyev** [comment]
should be closed by #321. Please give it a go.

**Comment by tsiq-karold** [comment]
It is, thanks!
